operator/option/config.go

```diff
@@ -147,9 +147,9 @@ const (
 	// AzureResourceGroup is the resource group of the nodes used for the cluster
 	AzureResourceGroup = "azure-resource-group"
 
-	// UserAssignedIdentityName is the name of the user assigned identity used
+	// UserAssignedIdentityID is the id of the user assigned identity used
```

Comment:
While we're at it, can you rename this variable to "AzureUserAssignedIdentityID"?

Reply:
Updated!